### PR TITLE
Support "Forwarded" header too

### PR DESCRIPTION
**Pull request by bschaepper**
Since there is a standardized replacement for `X-Forwarded-Proto`, the new [`Forwarded` header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Forwarded), we should probably support this too.

This change adds a check for the new header, after all preexisting checks for `issecure`. This allows to set secure cookies when a `Forwarded: proto=https` header is present.

### CODE REVIEW

**Comment by dougwilson** [comment]
Hello, and thank you for your pull request. The reading of x-forwarded-for is deprecated in this module and will be removed in next major version; instead `req.secure` is preferred, as it can be programmed to use any header name. Adding this directory is unfortunately tricky, as servers who are using x-forwarded-for at the gateway are unlikely to have filtered out the forwarded header to the backend.

Basically a duplicate request of #872

**Comment by bschaepper** [comment]
Hi @dougwilson, oh I wasn't aware of that. Looks like a better solution yes, it felt a bit clumsy to add indeed.

**Comment by dougwilson** [comment]
Haha, yea. Also there are some plans to make the proxy stuff more configurable and pluggable directly in Express. I'm glad to see the Frowarded header start to ger adoped, it's about time :) Express direcrly only supported the x- for all this time because there was practically no reason to otherwise, lol.
